driverless/src/main/java/com/upgrad/hirewheels/config/WebConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.upgrad.hirewheels.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class WebConfig implements WebMvcConfigurer {
+    @Override
+    public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**")
+                .allowedOrigins("http://localhost:4200")
+                .allowedMethods("GET", "POST", "PUT", "DELETE", "HEAD", "OPTIONS")
```

Comment:
I believe, this will not allow JWT tokens in the header as we have not specified it explicitly. Please add permissions for jwt-token as well.

`        registry.addMapping("/**")
                .allowedOrigins("http://localhost:4200")
                .allowedMethods("GET", "POST", "PUT", "DELETE", "HEAD", "OPTIONS")
                .allowedHeaders("Content-Type", "Date", "Total-Count", "loginInfo", "x-access-token")
                .exposedHeaders("Content-Type", "Date", "Total-Count", "loginInfo" , "x-access-token")
                .maxAge(3600);`